pkg/models/tenant/tenant.go

```diff
@@ -806,21 +805,41 @@ func (t *tenantOperator) Auditing(user user.Info, queryParam *auditingv1alpha1.Q
 	}
 
 	namespaceCreateTimeMap := make(map[string]time.Time)
+
 	for _, ns := range iNamespaces {
-		namespaceCreateTimeMap[ns.Name] = ns.CreationTimestamp.Time
+		listEvts := authorizer.AttributesRecord{
```

Comment:
We'd better to add some comments here to elaborate that auditing will use the same authorization logic as events to decide if user could see an auditing event